chapters/fa/chapter4/1.mdx

```diff
@@ -0,0 +1,23 @@
+<div dir="rtl">
+# هاب هاگینگ‌فیس
+
+[هاب هاگینگ‌فیس](https://huggingface.co/) –- وب‌سایت اصلی ما –- پلت‌فرمی مرکزی است که به همه امکان مشارکت، کشف و استفاده از جدیدترین و بهترین مدل‌ها و دیتاست‌ها را می‌دهد. این پایگاه، طیف گسترده‌ای از مدل‌ها را در خود جای داده، که بالغ بر ۱۰ هزار مورد از آن‌ها در دسترس عموم قرار دارد. در این فصل بر روی مدل‌ها متمرکز شده و در فصل ۵ نیز نگاهی به دیتاست‌ها خواهیم داشت.
+
+مدل‌های موجود در هاب، محدود به ترنسفورمرهای هاگینگ‌فیس یا حتی پردازش زبان طبیعی نیستند. از جمله آنها می‌توان مدل‌هایی از قبیل [Flair](https://github.com/flairNLP/flair) و [AllenNLP](https://github.com/allenai/allennlp) برای پردازش زبان طبیعی، [Asteroid](https://github.com/asteroid-team/asteroid) و [pyannote](https://github.com/pyannote/pyannote-audio) برای پردازش گفتار، و [timm](https://github.com/rwightman/pytorch-image-models) را برای پردازش تصویر برشمرد.
+
+هر یک از این مدل‌ها به عنوان مخزنی از Git میزبانی می‌شوند که این امر، امکان نسخه‌بندی[^1] و قابلیت بازتولید[^2] را فراهم می‌نماید. به‌اشتراک‌گذاشتن مدل در پایگاه، به معنای باز کردن آن به روی جامعه کاربران و در دسترس قرار دادن آن برای افرادی است که می‌خواهند به راحتی از آن استفاده کنند؛ در نتیجه، ضرورت اینکه افراد مدل را خودشان به تنهایی آموزش دهند، برطرف شده و هم‌رسانی و استفاده از آن تسهیل می‌شود.
```

Comment:
I'll go ahead with the convention the guide itself uses for "گذاری" postfix (by utilizing half-space on all occasions). This will be the final form: به اشتراک‌گذاری
Regarding "هم‌رسانی", it seems to me it looks more like "همدلی" (which is also exemplified in the guide), rather than "تورم‌زا". So, I'll go with "همرسانی".
On that note, should we also include these for "share / sharing" inside G?
![gozaari](https://user-images.githubusercontent.com/24595790/164561040-82ff451c-f0a3-4ccd-bd27-20354679059d.jpg)

Reply:
Agreed on both. Also, yes lets include share with both these equivalents and their correct spellings in the glossary.

Reply:
> Agreed on both. Also, yes lets include share with both these equivalents and their correct spellings in the glossary.
> 
> Really need to sleep now Hamed Jan, you go ahead I'll be with you again in the morning. Apologies in advance for late replies.

❤️